api/nodes/serializers.py

```diff
@@ -264,8 +264,7 @@ def update(self, node, validated_data):
         auth = get_user_auth(self.context['request'])
         old_tags = set([tag._id for tag in node.tags])
         if 'tags' in validated_data:
-            current_tags = set(validated_data.get('tags'))
-            del validated_data['tags']
```

Comment:
It seems like it is necessary for the check ~L279 to be correct, but I'm not sure.